### DISCUSSION TITLE
local host site blank white screen

### DISCUSSION THREAD

**Discussion by jhogan12-lab** [datalust/seq-tickets#2041] [Help] [answered]
Good day Seq team,

I have Seq installed on an RDP with a windows service pointed to the logger. The logging installation was successful and it is running as a windows service. When I navigate to the local host site, it does not load and only gives a white screen. It does not load any further.

**Comment by jhogan12-lab** [comment, marked as answer]
nevermind, I was using an incorrect browser version

**Reply by nblumhardt** [reply]
Great! Glad you found the issue, thanks for the follow-up :+1:
